gnuradio-runtime/include/gnuradio/block.h

```diff
@@ -545,9 +545,9 @@ class GR_RUNTIME_API block : public basic_block
      * \brief Allocate the block_detail and necessary output buffers for this
      * block.
      */
-    void allocate_detail(int ninputs,
-                         int noutputs,
-                         const std::vector<int>& downstream_max_nitems_vec,
+    void allocate_detail(unsigned int ninputs,
+                         unsigned int noutputs,
+                         const std::vector<unsigned int>& downstream_max_nitems_vec,
```

Comment:
Here, for example, I know it's preference, but I don't like that space in the template param.